config/core/deployments/domainmapping-controller.yaml

```diff
@@ -91,14 +91,14 @@ spec:
             port: probes
             scheme: HTTP
           periodSeconds: 5
-          failureThreshold: 5
+          failureThreshold: 3
         readinessProbe:
           httpGet:
             path: /readiness
             port: probes
             scheme: HTTP
           periodSeconds: 5
-          failureThreshold: 5
+          failureThreshold: 6
```

Comment:
I think we want the `failureThreshold` for `readinessProbes` to be _lower_ than the `failureThreshold` for `livenessProbes`.  Otherwise, we're choosing to terminate the container _before_ we remove it from the active set (which was the issue that Julz filed).